portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,37 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div class="cover">
+      <h>Annie Yang</h>
+    </div>
+    <div class="topnav" id="myTopNav">
```

Comment:
There are a couple more classes that I don't think have been implemented in this PR - "topnav", "active", "about", and "project". I would either remove them from the HTML file (and then add them properly in a future PR), or add them to the CSS in this PR.

Reply:
yep i will add them into the PR, thanks!

Reply:
Done! added the CSS into the PR.